src/DataGrid.tsx

```diff
@@ -483,6 +490,16 @@ function DataGrid<R, SR, K extends Key>(
   function handleColumnResize(column: CalculatedColumn<R, SR>, width: number | 'max-content') {
```

Comment:
Can we send `${width}px` to `onColumnResize` in `HeaderCell`?
```suggestion
  function handleColumnResize(column: CalculatedColumn<R, SR>, width: string) {
```